socialdistribution/templates/base.html

```diff
@@ -6,6 +6,17 @@
   </head>
 
   <body>
-    <div id="content">{% block content %}{% endblock %}</div>
+    <div id="content">
+      <!-- https://www.youtube.com/watch?v=q4jPR-M0TAQ&list=PL-osiE80TeTtoQCKZ03TU5fNfx2UY6U4p&index=6 -->
+      <!-- Will give a notification when register successfully -->
+      {% if messages%}
+            {%for message in messages%}
+            <div class="alert alert-{{message.tags}}"> 
+                {{message}}
+            </div>
+            {%endfor%}
+            {%endif%}
+      {% block content %}{% endblock %}
+    </div>
```

Comment:
I think we should put this content in the `login.html` file instead. If we have it in the base file it will display the message on every page in the app, and I think we want to keep it to the login page only.

Reply:
Sounds great